openpype/hosts/houdini/api/plugin.py

```diff
@@ -84,3 +90,109 @@ def process(self):
                 OpenPypeCreatorError,
                 OpenPypeCreatorError("Creator error: {}".format(er)),
                 sys.exc_info()[2])
+
+
+@six.add_metaclass(ABCMeta)
```

Comment:
There is no need to add `ABCMeta` it is already in `NewCreator`.

Reply:
I think it is more readable to be explicit - that by simple look at the code you know you shouldn't use that class directly.